### ISSUE TITLE
Plugin is trying to read the theme configuration before it is loaded

### ISSUE THREAD

**Issue by nixilla** [cowsay-blog/hexo-webpack#1]
It looks like the theme `_config.yml` is loaded after plugin is loaded, therefore plugin does not have access to the `theme/<theme_name>/_config.yml`.

Debug info:
```
11:52:15.814 DEBUG Plugin loaded: hexo-generator-archive
11:52:15.815 DEBUG Plugin loaded: hexo-generator-category
11:52:15.817 DEBUG Plugin loaded: hexo-generator-index
11:52:15.818 DEBUG Plugin loaded: hexo-generator-tag
11:52:15.820 DEBUG Plugin loaded: hexo-renderer-ejs
11:52:15.858 DEBUG Plugin loaded: hexo-server
11:52:15.862 DEBUG Plugin loaded: hexo-renderer-marked
11:52:15.862 DEBUG Plugin loaded: hexo-renderer-stylus
{} // output from: console.log(_.get(hexo, 'theme.config'));
{ instance: undefined, theme: undefined }
11:52:16.136 DEBUG Plugin loaded: hexo-webpack
11:52:16.139 DEBUG Loading database.
11:52:16.146 INFO  Start processing
11:52:16.157 DEBUG Theme config loaded.
11:52:16.158 DEBUG Processed: _config.yml // actual moment when the theme config is loaded
```

**Comment by momocow** [comment]
@nixilla Thanks for reporting the issue. I've found the 🐛 and a patch will be released soon.

**Comment by momocow** [comment]
Fixed in `v1.0.0`, as a formal release. (Previous `v0.1` is for testing.)
https://github.com/cowsay-blog/hexo-webpack/releases/tag/v1.0.0
